src/pages/settings/support/users/_id.vue

```diff
@@ -97,36 +319,101 @@ export default {
       console.error(err);
       this.$router.replace({ name: 'settings-support' });
     }
-    this.state = 'done';
     // Set default to  1 for new subscription
     const { quantity = 1 } =
       this.getCivicSupportingUserInfo(this.authorId) || {};
     this.selectedQuantity = quantity;
+    this.state = this.getUserIsCivicLiker ? 'select-quantity' : 'new';
   },
   methods: {
     ...mapActions([
       'fetchCivicSupportingUsers',
+      'fetchUserInfo',
       'fetchUserSubscriptionInfo',
       'updateCivicSupportQuantity',
       'removeCivicSupportUser',
     ]),
+
+    getPriceEmoji,
+
+    async fetchLikerInfo() {
+      try {
+        this.isLoading = true;
+        if (this.authorId && !this.getUserInfoById(this.authorId)) {
+          await this.fetchUserInfo(this.authorId);
+        }
+        const authorData = this.getUserInfoById(this.authorId) || {};
+        this.displayName = authorData.displayName;
+        this.avatarUrl = authorData.avatar;
+        this.isCivicLiker =
+          authorData.isCivicLikerTrial || authorData.isSubscribedCivicLiker;
+      } catch (err) {
+        // eslint-disable-next-line no-console
+        console.error(err);
+      } finally {
+        this.isLoading = false;
+      }
+    },
+
+    confirmQuantity() {
+      this.state = this.getUserIsCivicLiker ? 'confirm' : 'new';
+    },
+
+    newSubscription() {
+      this.$router.push({
+        name: `civic-register-stripe`,
+        query: {
+          from: this.authorId,
+          civic_liker_version: 2,
```

Comment:
We would remove `civic_liker_version` to default everyone to `2` when public release

Reply:
Then, will remove this query after release